packages/sdk/test/evm/custom.test.ts

```diff
@@ -298,6 +298,26 @@ describe("Custom Contracts", async () => {
     expect(nfts[0].metadata.name).to.eq("Custom NFT");
   });
 
+  it("should transfer erc721", async () => {
```

Comment:
Should also add some test cases to check that the error messages work properly with `Transaction`